draft-ietf-modpod-group-processes.md

```diff
@@ -645,3 +646,23 @@ The moderator team's task consists of
 subjective judgement calls. Behaviors not listed here might require
 moderation, and it is not possible to write a complete list of all such
 behaviors.
+
+# The risks of moderation and (self) censorship
```

Comment:
I think the term `censorship` is entirely inappropriate here.

Reply:
I disagree.  That is in fact the risk.